app/scripts/e2e-run-all-flows.sh

```
@@ -3,31 +3,159 @@
 # Requires a prior E2E app build: pnpm --filter openhuman-app test:e2e:build
 #
-set -euo pipefail
+# Each spec runs to completion regardless of prior failures; a pass/fail
+# summary is printed at the end and the script exits non-zero if any spec
+# failed. (Previously `set -e` caused the first failure to abort the run
+# and made the terminal appear to crash.)
+#
+set -uo pipefail
 
 APP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
 cd "$APP_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)